_data/banking.yml

```diff
@@ -351,6 +351,13 @@ websites:
       software: Yes
       doc: https://www.handelsbanken.se/shb/INeT/ICentSv.nsf/Default/qAE99695C4C1D033DC12579120036EB43?Opendocument
 
+    - name: Holvi
+      url: https://www.holvi.com/
```

Comment:
@2factorauth/collaborators should we keep the URL or use the one that the current one redirects to?